SPECS/softhsm/softhsm.spec

```diff
@@ -1,29 +1,24 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-#global prever rc1
-#global prerelease yes
-
-Summary: Software version of a PKCS#11 Hardware Security Module
-Name: softhsm
-Version: 2.6.1
-Release: 5%{?dist}
-License: BSD
-Url: http://www.opendnssec.org/
-Source: http://dist.opendnssec.org/source/%{?prever:testing/}%{name}-%{version}.tar.gz
-Source1: http://dist.opendnssec.org/source/%{?prever:testing/}%{name}-%{version}.tar.gz.sig
```

Comment:
I think this file also needs to be deleted. Also, the signatures should probably be updated as well (can't see the contents on GitHub, but there seems to be no changes in the signatures JSON).

Reply:
Updated the signature file. The .sig file is not on github